nixos/modules/services/web-servers/caddy.nix

```diff
@@ -6,90 +6,101 @@ let
   cfg = config.services.caddy;
   configFile = pkgs.writeText "Caddyfile" cfg.config;
 in
-{
-  options.services.caddy = {
-    enable = mkEnableOption "Caddy web server";
+  {
```

Comment:
Ah, I didn't catch that. Thank you. Should've looked more carefully. Fixed now, squashed with the original commit.